### PR TITLE
refactor(CurriculumCard): new prop to set height to 100%

### PR DESCRIPTION
**Pull request by aionate0812**
### Overview

New prop to set the height of the card to 100%

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [Reach-1065](https://codecademy.atlassian.net/browse/REACH-1065)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.36-alpha.624ae3.0<br/>@codecademy/gamut-labs@19.10.2-alpha.624ae3.0<br/>@codecademy/styleguide@41.10.6-alpha.624ae3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60cba7901408bd00b784bf33--gamut-preview.netlify.app>https://60cba7901408bd00b784bf33--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60cba7901408bd00b784bf33>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.45-alpha.267c83.0<br/>@codecademy/gamut-labs@19.11.1-alpha.267c83.0<br/>@codecademy/styleguide@41.10.15-alpha.267c83.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60d202602d18a10c4d85bedb--gamut-preview.netlify.app>https://60d202602d18a10c4d85bedb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60d202602d18a10c4d85bedb>Deploy Logs</a></p>

**Review by christian-dinh — APPROVED** [review]
🔪

**Review by aresnik11 — COMMENTED** [review]
copying my slack message here for visibility:
i'm not sure we want to truncate the titles here. i think we instead may want to just stretch the other cards in that row so the row is even, or at least thats what it looks like we were doing in the old catalog (top) vs portal app (bottom)
Screen Shot 2021-06-22 at 11 33 11 AM
Screen Shot 2021-06-22 at 11 33 27 AM

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
+1, what Amy said :)

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.45-alpha.52fb74.0<br/>@codecademy/gamut-labs@19.11.1-alpha.52fb74.0<br/>@codecademy/styleguide@41.10.15-alpha.52fb74.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60d23e68ee3325120412bb6b--gamut-preview.netlify.app>https://60d23e68ee3325120412bb6b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60d23e68ee3325120412bb6b>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.45-alpha.54b8ac.0<br/>@codecademy/gamut-labs@19.11.1-alpha.54b8ac.0<br/>@codecademy/styleguide@41.10.15-alpha.54b8ac.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60d24904d28bda00aac9c0f3--gamut-preview.netlify.app>https://60d24904d28bda00aac9c0f3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60d24904d28bda00aac9c0f3>Deploy Logs</a></p>

File: packages/gamut-labs/src/CurriculumCard/index.tsx [hunk 1]
```diff
@@ -53,6 +55,7 @@ export const CurriculumCard: React.FC<CurriculumCardProps> = ({
       variant={boxVariant ?? 'white'}
       shadow="medium"
       position="relative"
+      height={isFullHeight ? '100%' : ''}
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/CurriculumCard/index.tsx:58]: I'm averse to adding assorted sizing props to shared components. Their complexity can get pretty intense.

Instead, have you tried using CSS properties in the consuming app to make things stretch? Like `align-items: stretch` / `justify-content: stretch` type of thing?

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]

File: packages/gamut-labs/src/CurriculumCard/index.tsx (hunk 1 above)
Comment (aionate0812) [reply packages/gamut-labs/src/CurriculumCard/index.tsx:58]: Trying that [here](https://github.com/codecademy-engineering/portal-app/pull/170)

**Pull request closed without merging**